example_14/hardware/rrbot_sensor_for_position_feedback.cpp

```diff
@@ -261,23 +251,22 @@ hardware_interface::return_type RRBotSensorPositionFeedback::read(
   last_timestamp_ = current_timestamp;
 
   // START: This part here is for exemplary purposes - Please do not copy to your production code
-  RCLCPP_INFO(rclcpp::get_logger("RRBotSensorPositionFeedback"), "Reading...");
+  std::stringstream ss;
+  ss << "Reading..." << std::endl;
 
   // Simulate RRBot's movement
   measured_velocity = *(rt_incomming_data_ptr_.readFromRT());
   if (!std::isnan(measured_velocity))
   {
     last_measured_velocity_ = measured_velocity;
   }
-  RCLCPP_INFO(
-    rclcpp::get_logger("RRBotSensorPositionFeedback"), "Got measured velocity %.5f",
-    measured_velocity);
   hw_joint_state_ += (last_measured_velocity_ * duration.seconds()) / hw_slowdown_;
-  RCLCPP_INFO(
-    rclcpp::get_logger("RRBotSensorPositionFeedback"), "Got state %.5f for joint '%s'!",
-    hw_joint_state_, info_.joints[0].name.c_str());
 
-  RCLCPP_INFO(rclcpp::get_logger("RRBotSensorPositionFeedback"), "Joints successfully read!");
+  ss << std::fixed << std::setprecision(2) << std::endl;
```

Comment:
Otherwise we end up with an empty line between "Reading..." and the measured velocity.
```suggestion
  ss << std::fixed << std::setprecision(2);
```